examples/react/src/stories/csf.stories.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { linkTo } from '@storybook/addon-links';
```

Comment:
Does the example need these 2 imports? 🤔

Reply:
Fixed!